bin/setup

```diff
@@ -9,8 +9,8 @@ def system!(*args)
 end
 
 FileUtils.chdir APP_ROOT do
-  # This script is a way to setup or update your development environment automatically.
-  # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
+  # This script is a way to set up or update your development environment automatically.
+  # This script is idempotent, so that you can run it at any time and get an expectable outcome.
```

Comment:
this isn't important to include in rails 7 upgrade, let's avoid makin irrelevant changes

Reply:
I think it should be fine? Since it's merely English word fixup, and if we let it be like the previously it will end up not being attended anymore